bin/main.ml

```diff
@@ -189,6 +188,11 @@ let find_root () =
 let package_name =
   Arg.conv ((fun p -> Ok (Package.Name.of_string p)), Package.Name.pp)
 
+let path_arg =
```

Comment:
What about:

```ocaml
module Arg = struct
  include Arg
  let file = ...
end
```

? 

Just to make sure we don't use the wrong combinator.

Reply:
Sure, that makes sense. And the original args issue?